docs/_data/navigation.yml

```diff
@@ -2,8 +2,14 @@
    link: About
```

Comment:
The About link needs to go to `/#About` (I think), since it otherwise doesn't work on subpages.

Reply:
this should work on deployed version as we discussed on chat. (About page is not being used)
still if this doesn't work please let us know.